File: DSA/AadrikaBehl_590012053/Day1Q3.c
#include <stdio.h>

int main() 
{
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);

    int arr[n];

    printf("Enter %d integers:\n", n);
    for(int i = 0; i < n; i++) 
    {
        scanf("%d", &arr[i]);
    }

    int sum = 0;
    for(int i = 0; i < n; i++) 
    {
        sum += arr[i];
    }

printf("Sum of array elements: %d\n", sum);

return 0;
}

File: DSA/Muskan_590015043/Day1_Q3.py
#A fundamental array operation that calculates the total sum of all elements.
#INPUT THE ARRAY
n = int(input("Enter the number of elements in the array: "))
ls = [] #Initialise empty list
for i in range(n):
    ls.append(int(input()))
sum = 0 #Update variable sum initiallys et to 0
for element in ls:
    sum += element #Iterates through each elemnt and adds to the variable sum
print(sum)